lib/Controller/RoomController.php

```diff
@@ -1363,4 +1369,91 @@ protected function demoteGuestFromModerator(Room $room, string $sessionId): Data
 
 		return new DataResponse();
 	}
+
+	/**
+	 * Get bridge information on one room
+	 *
+	 * @NoAdminRequired
+	 *
+	 * @return DataResponse
+	 */
+	public function getBridgeOfRoom(string $token): DataResponse {
+		try {
+			$room = $this->manager->getRoomForParticipantByToken($token, $this->userId, true);
+			try {
+				$participant = $room->getParticipant($this->userId);
+				if ($participant->hasModeratorPermissions()) {
+					$this->bridgeManager->checkBridge($token);
+					$bridge = $this->bridgeManager->getBridgeOfRoom($token);
+					return new DataResponse($bridge);
+				} else {
+					return new DataResponse([], Http::STATUS_FORBIDDEN);
+				}
+			} catch (ParticipantNotFoundException $e) {
+				return new DataResponse([], Http::STATUS_NOT_FOUND);
+			}
+		} catch (RoomNotFoundException $e) {
+			return new DataResponse([], Http::STATUS_NOT_FOUND);
+		}
+	}
+
+	/**
+	 * Edit bridge information on one room
+	 *
+	 * @NoAdminRequired
+	 *
+	 * @return DataResponse
+	 */
+	public function editBridgeOfRoom(string $token, bool $enabled, array $parts = []): DataResponse {
+		try {
+			$room = $this->manager->getRoomForParticipantByToken($token, $this->userId, true);
+			try {
+				$participant = $room->getParticipant($this->userId);
+				if ($participant->hasModeratorPermissions()) {
+					try {
+						$success = $this->bridgeManager->editBridgeOfRoom($token, $enabled, $parts);
+					} catch (ImpossibleToKillException $e) {
+						return new DataResponse(['error' => $e->getMessage()], Http::STATUS_NOT_ACCEPTABLE);
+					}
+					return new DataResponse($success);
+				} else {
+					return new DataResponse([], Http::STATUS_FORBIDDEN);
+				}
+			} catch (ParticipantNotFoundException $e) {
+				return new DataResponse([], Http::STATUS_NOT_FOUND);
+			}
+		} catch (RoomNotFoundException $e) {
+			return new DataResponse([], Http::STATUS_NOT_FOUND);
+		}
+	}
+
+	/**
+	 * Delete bridge of one room
+	 *
+	 * @NoAdminRequired
+	 *
+	 * @return DataResponse
+	 */
+	public function deleteBridgeOfRoom(string $token): DataResponse {
+		try {
+			$room = $this->manager->getRoomForParticipantByToken($token, $this->userId, true);
+			try {
+				$participant = $room->getParticipant($this->userId);
+				if ($participant->hasModeratorPermissions()) {
```

Comment:
If this stays in talk, use `@RequireLoggedInModeratorParticipant` annotation instead.